openreferee_server/notify.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+import os
+import threading
+from requests.exceptions import HTTPError, ConnectionError, Timeout, RequestException
+from .operations import setup_requests_session
+
+
+class NotifyService:
+    def __init__(self, url=None, token=None, logger=None) -> None:
+        self.url = url
+        self.token = token
+        self.logger = logger
+        self.session = None
+
+    def log_error(self, *args) -> logging.Logger:
```

Comment:
```suggestion
    def log_error(self, *args):
```

I don't think return type annotations are particularly useful for this, and this one was wrong as well: None of the logging methods actually return anything.